internal/apiserver/draft.go

```diff
@@ -60,6 +90,17 @@ func (a *App) getMyDraft(res http.ResponseWriter, req *http.Request) error {
 }
 
 func (a *App) createMyDraft(res http.ResponseWriter, req *http.Request) error {
+	ctx := req.Context()
+
+	// Limit number of drafts
+	drafts, err := a.getDraftsMetadataForUser(ctx)
+	if err != nil {
+		return err
+	}
+	if len(drafts) >= a.config.Cache.DraftsLimit {
```

Comment:
maybe `>` would be better so limit means how many drafts you can have

Also we can move this variable from cache subconfig as it is used here not in cache